app/ts/background/background.ts

```diff
@@ -102,6 +105,7 @@ export const simulateGovernanceContractExecution = async (pendingTransaction: Pe
 		}
 		return await visualizeSimulatorState(governanceContractSimulationState, ethereum)
 	} catch(error) {
+		console.warn(error)
 		if (error instanceof Error) return returnError(error.message)
 		throw error
```

Comment:
Why do you throw an error if it isn't an `instanceof Error` rather than returning it like all of the other errors?